packages/core/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--
+  Thanks for contributing!
+
+  Provide a description of your changes below and a general summary in the title
+
+  Please look at the following checklist to ensure that your PR can be accepted quickly:
```

Comment:
Is this in the correct folder?

Reply:
I forgot to move the rest of this folder 🥲 Will move it now